pathname/index.md details (#21084)
mdn/content: files/en-us/web/api/location/pathname/index.md

{{ApiRef("Location")}}

The **`pathname`** property of the {{domxref("Location")}}

> interface is a string containing the path of the URL for the location, which will be the empty string if there is no path. If not empty, it contains an initial '/' followed by the path of the URL, not including the query string or fragment.

Reviewer: ```suggestion
interface is a string containing the path of the URL for the location. If there is no path, `pathname` will be empty: otherwise, `pathname` contains an initial '/' followed by the path of the URL, not including the query string or fragment.
```

(I thought "If not empty" was a bit clumsy here. Please feel free to argue though if you don't like the suggested change :)
